app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java

```diff
@@ -18,6 +18,9 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static final int defaultFontSize=120;
+    private static final int scale=8;
```

Comment:
final vars should always be named in capital letters also move it to the top with other final vars, also add space before and after the `=`

Reply:
ok np